packages/core/src/stateUtils.ts

```diff
@@ -1547,6 +1548,21 @@ export function resolveMicroTransition<
     delete history.history;
   }
 
+  nextState.actions.forEach((action) => {
```

Comment:
This introduces the previously existing behavior in V4 where invoked actors were present in `state.children` immediately (while possibly being `deferred`)